apis/duck/typed.go

```diff
@@ -46,6 +46,11 @@ var _ InformerFactory = (*TypedInformerFactory)(nil)
 
 // Get implements InformerFactory.
 func (dif *TypedInformerFactory) Get(gvr schema.GroupVersionResource) (cache.SharedIndexInformer, cache.GenericLister, error) {
+	// Avoid error cases, like resource not exists.
```

Comment:
```suggestion
	// Avoid error cases, like the GVR does not exist.
```
I'd also expand the comment to say that it's not a full check, that some RBACs might sneak by, but the window is very small.